lib/Models/Internationalization.ts

```diff
@@ -121,18 +120,13 @@ class Internationalization {
         defaultNS: "languageOverrides",
         fallbackNS: "translation",
 
-        resources: {
-          en: {
-            translation: translationEN
-          },
-          fr: {
-            translation: translationFR
-          }
-        },
-
         backend: Object.assign(
           {
-            loadPath: "/languages/{{lng}}/{{ns}}.json",
+            loadPath: function loadPath(lngs: string[], namespaces: string[]) {
+              return namespaces[0] === "translation"
+                ? `${terriajsResourcesBaseUrl}languages/{{lng}}/{{ns}}.json`
+                : "languages/{{lng}}/{{ns}}.json";
+            },
```

Comment:
Switch translation loading from importing JSON files (including translation in bundle) to loading on demand.